src/repositories/qfRoundRepository.ts

```diff
@@ -22,7 +22,9 @@ export const findAllQfRounds = async ({
     query.where('slug = :slug', { slug });
   }
   if (activeOnly) {
-    query.andWhere('"isActive" = true');
+    query
+      .andWhere('"isActive" = true')
+      .cache('findQfRounds-activeOnly', 1000 * 60 * 60);
```

Comment:
can we take this from config?

Reply:
Agreed @RamRamez lets move this to a config. As most caches, so its configurable, but leave this as the default value in the constant.

Reply:
Good idea!
I changed it to fetch from env var, and set 2 min caching on staging for easier testing.